### PR TITLE
Fix hotkeys blocking comments

### PR DESCRIPTION
**Pull request by charlesBochet**
Hot fixes to make usable:
- fix create person failing because empty email and phoneNumber do not match validation format
- fix right drawer not taking the focus

### CODE REVIEW

File: server/src/core/@generated/person/person-create-many-company.input.ts [hunk 1]
```diff
@@ -20,11 +20,11 @@ export class PersonCreateManyCompanyInput {
     lastName!: string;
 
     @Field(() => String, {nullable:false})
-    @Validator.IsEmail()
+    @Validator.IsString()
```
Comment (emilienchvt) [inline server/src/core/@generated/person/person-create-many-company.input.ts:23]: Could `@IsOptional()` or another `class-validator` decorator be a more secure alternative ?

File: front/src/modules/ui/layout/right-drawer/components/RightDrawer.tsx [hunk 2]
```diff
@@ -1,6 +1,8 @@
+import { useEffect } from 'react';
 import styled from '@emotion/styled';
 import { useRecoilState } from 'recoil';
 
+import { captureHotkeyTypeInFocusState } from '@/hotkeys/states/captureHotkeyTypeInFocusState';
```
Comment (emilienchvt) [inline front/src/modules/ui/layout/right-drawer/components/RightDrawer.tsx:5]: Is this specific focus behaviour tested ?

Comment (charlesBochet) [reply front/src/modules/ui/layout/right-drawer/components/RightDrawer.tsx:5]: no, and we are going to remove it completely on Monday as we merge Lucas' refacto :(

File: server/src/core/@generated/person/person-create-many-company.input.ts (hunk 1 above)
Comment (charlesBochet) [reply server/src/core/@generated/person/person-create-many-company.input.ts:23]: This is to be discussed IMO. In the current design, a non-set email is the "" empty string. IsOptional() + IsEmail() means `undefined | email`. What we would like is `"" | email`
These validations were added too quickly!

**Review by emilienchvt — APPROVED** [review]
Sounds good 👌

**Pull request merged**